serverless-logic/serverless-logic-managing-services/serverless-logic-openAPI-callback.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="serverless-logic-openAPI-callback_{context}"]
+= Understanding OpenAPI callback and callback state
+:context: serverless-logic-openAPI-callback
+include::_attributes/common-attributes.adoc[]
+
+You can use OpenAPI callbacks and Callback states to handle actions that require waiting for events. Both methods perform an action and wait for an event to continue the workflow, but they achieve this differently. Both are effective for `fire & wait-for-result` operations.
+
+OpenAPI callback manages asynchronous, external responses from other services. Your service makes a request to another service. That service responds to specific events by sending a request back to your callback URL. Whereas callback state, executes an action that triggers an external service to send a CloudEvent back to the workflow. The workflow performs an action and waits for a CloudEvent to return, which then resumes the workflow.
+
```

Comment:
I think that if possible, we should try to copy the example snippets from the original guide. This will be very helpful for people to understand and use.

Reply:
As per the discussion, we are removing this part from the PR.

Reply:
Still in the PR, remove it agreed please